packages/stream_chat_flutter_core/pubspec.yaml

```diff
@@ -10,8 +10,8 @@ environment:
   flutter: ">=1.17.0"
 
 dependencies:
-  stream_chat: ^0.2.23+3
-  rxdart: ^0.24.1
+  stream_chat:
+    path: ../dart_client
```

Comment:
Consider using the version number instead, melos boostrap automatically handles linking.